.circleci/config.yml

```diff
@@ -381,11 +381,9 @@ workflows:
           type: approval
```

Comment:
we could maybe rename from `hold` to `hold-firebase-tests`, so it's more clear what the hold does